std/machines/large_field/rotate.asm

```diff
@@ -0,0 +1,66 @@
+use std::utils::unchanged_until;
+use std::utils::cross_product;
+use std::convert::int;
+
+// Rotate for single bytes using an exhaustive table
+// TODO this way, we cannot prove anything that rotates by more than 31 bits.
```

Comment:
```suggestion
/// Rotate for single bytes using an exhaustive table
/// TODO this way, we cannot prove anything that rotates by more than 31 bits.
```

Reply:
Yes, that would have a better explanation